src/dp/score_vector2.h

```diff
@@ -50,8 +50,6 @@ Author: Benjamin Buchfink
  #include "../sse2neon.h"
 #endif
  
--#include <smmintrin.h>
```

Comment:
It is good that I added CI!
I tested #8 on ARM64 and made a copy/paste error while moving the changes to my dev machine (x86_64) ...

Reply:
@aquaskyline Friendly ping!
Currently the build is broken due to this line.